### ISSUE TITLE
optional-chaining

### ISSUE THREAD

**Issue by yotam707** [Khan/flow-to-ts#182]
Hi Is there a way to when flow-to-ts run to avoid changes in optional-chaining parameters? 
this: 
```
  ${props => props.theme?.components?.BillPaymentReview?.PaymentInfoContainer}
```
turned into this: 
```
 ${(
    props // Auto generated from flowToTs. Please clean me!
  ) =>
    ((props.theme === null || props.theme === undefined ? undefined : props.theme.components) ===
      null ||
    (props.theme === null || props.theme === undefined ? undefined : props.theme.components) ===
      undefined
      ? undefined
      : (props.theme === null || props.theme === undefined ? undefined : props.theme.components)
          .BillPaymentReview) === null ||
    ((props.theme === null || props.theme === undefined ? undefined : props.theme.components) ===
      null ||
    (props.theme === null || props.theme === undefined ? undefined : props.theme.components) ===
      undefined
      ? undefined
      : (props.theme === null || props.theme === undefined ? undefined : props.theme.components)
          .BillPaymentReview) === undefined
      ? undefined
      : ((props.theme === null || props.theme === undefined
          ? undefined
          : props.theme.components) === null ||
        (props.theme === null || props.theme === undefined ? undefined : props.theme.components) ===
          undefined
          ? undefined
          : (props.theme === null || props.theme === undefined ? undefined : props.theme.components)
              .BillPaymentReview
        ).PaymentInfoContainer}
```

**Comment by k4b7** [comment]
I get a syntax error in the https://flow-to-ts.netlify.app/.

**Comment by k4b7** [comment]
We'll probably need to add a babel syntax plugin for this while making sure that the transform plugin is not included.

**Comment by k4b7** [comment]
It turns out prettier was the thing that wasn't causing the error.  I've upgrade prettier in #212 to solve the issue.  https://deploy-preview-212--flow-to-ts.netlify.app/#code=Ly8gQGZsb3cKY29uc3QgYSA9IGAke3Byb3BzID0+IHByb3BzLnRoZW1lPy5jb21wb25lbnRzPy5CaWxsUGF5bWVudFJldmlldz8uUGF5bWVudEluZm9Db250YWluZXJ9YDsK&prettier=1&semi=1&singleQuote=0&tabWidth=4&trailingComma=all&bracketSpacing=0&arrowParens=avoid&printWidth=80&inlineUtilityTypes=0 shows that optional chaining is working in that PR.

**Comment by k4b7** [comment]
The generate code should now be
```
{props => props.theme?.components?.BillPaymentReview?.PaymentInfoContainer}
```
Please give it a try on `master`.  I'll be publishing a new version as soon as I fix an issue with comments that I introduced recently.

**Comment by k4b7** [comment]
I've published v0.2.0 which contains the fix.
